lib/brdocs/cpf/generator.ex

```diff
@@ -23,6 +23,7 @@ defmodule BrDocs.CPF.Generator do
         %BrDocs.BrDoc{kind: :cnpj, value: "111.444.777-35"}
 
   """
+  @spec generate(keyword()) :: BrDocs.BrDoc.t()
```

Comment:
```suggestion
  @spec generate(keyword(boolean())) :: BrDocs.BrDoc.t()
```
The value is always boolean, right?

Reply:
Yes, always. It also has a default `false`. I'll change in CNPJ generator as well.